protocol/storage/miner.go

```diff
@@ -748,18 +748,18 @@ func (sm *Miner) OnNewHeaviestTipSet(ts types.TipSet) {
 	}
 }
 
-func (sm *Miner) getProvingPeriodEnd() (*types.BlockHeight, error) {
+func (sm *Miner) getProvingPeriod() (*types.BlockHeight, *types.BlockHeight, error) {
 	res, err := sm.porcelainAPI.MessageQuery(
 		context.Background(),
 		address.Undef,
 		sm.minerAddr,
-		"getProvingPeriodEnd",
+		"isInProvingPeriod",
```

Comment:
Blocking: This looks like something from an interim attempt. LGTM if changed to `getProvingPeriod`.